tests/__init__.py

```diff
@@ -59,23 +59,15 @@ def setup_db(guarantee_empty=False):
 
 def _reset_db():
     engine = engine_from_config(SETTINGS, prefix='sqlalchemy.')
-    if engine.name == 'sqlite':
-        # Can't alembic downgrade sqlite because it can't do
-        # ALTER TABLE X DROP COLUMN
-        try:
-            os.remove(engine.url.database)
-        except OSError:
-            pass
-    elif engine.name == 'postgresql':
-        try:
-            engine.execute("DELETE FROM concept_note")
-            engine.execute("DELETE FROM note")
-            engine.execute("DELETE FROM concept_label")
-            engine.execute("DELETE FROM label")
-            command.downgrade(ALEMBIC_CONFIG, 'base')
-        except ProgrammingError:
-            """The tables may not exist if it's first time."""
+    try:
+        engine.execute("DELETE FROM concept_note")
+        engine.execute("DELETE FROM note")
+        engine.execute("DELETE FROM concept_label")
+        engine.execute("DELETE FROM label")
         command.downgrade(ALEMBIC_CONFIG, 'base')
+    except (ProgrammingError, OperationalError):
+        """The tables may not exist if it's first time."""
+    command.downgrade(ALEMBIC_CONFIG, 'base')
```

Comment:
lijkt mij een fout van mij om die downgrade niet in die except te steken. Anders loopt die nutteloos 2x als hij niet in de except komt.